### DISCUSSION TITLE
Auto tag resources in AWS using Cloud custodian & update the lastmodified resource tag with userid when any user update.

### DISCUSSION THREAD

**Discussion by Gautami007** [cloud-custodian/cloud-custodian#7367] [General]
Is it at all possible to filter by CloudTrail metadata?

I am figuring out how to do set a policy that will

1. Detect AWS resources that were created manually on the web console
2. Automatically add below tags to the resource
	a- Owner - should pick up value automatically "type: auto-tag-user"
	b- LastModified - should pick up value automatically "type: auto-tag-user"
3. Also, if someone update/modify the configuration of any resource example security group inbound/outbound rule, then it will only update in tag b "LastModified".

**Comment by kapilt** [comment]
1. use event filter to detect console
2. just use tag you can reference variables like event and current time {now} .. ie auto-tag user is specialized to work across different envs. but if you assume a console login with particular creds mechanism you already have unambigious ref to user in event.
3. to what value date time.. then use {now}
